packages/react/src/Accordion/AccordionSection.tsx

```diff
@@ -0,0 +1,68 @@
+/**
+ * @license EUPL-1.2+
+ * Copyright (c) 2023 Gemeente Amsterdam
+ */
+
+import { ChevronDown } from '@amsterdam/design-system-react-icons'
+import clsx from 'clsx'
+import { ForwardedRef, forwardRef, HTMLAttributes, PropsWithChildren, useContext, useId, useState } from 'react'
+import AccordionContext from './AccordionContext'
+import { getElement } from '../Heading/Heading'
```

Comment:
Should we name this better and add an index to Heading?
 
```suggestion
import { getHeadingElement } from '../Heading'
```

Reply:
I like the rename. I don't think we should add the function to the index of Heading though, we don't really need to expose this function to our consumers